packages/oidc/oidc.0.2.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: "Base package for working with OIDC"
+description: "Base functions and types to work with OpenID Connect."
+maintainer: ["ulrik.strid@outlook.com"]
+authors: ["Ulrik Strid"]
+license: "BSD-3-Clause"
+homepage: "https://ulrikstrid.github.io/ocaml-oidc"
+doc: "https://ulrikstrid.github.io/ocaml-oidc"
+bug-reports: "https://github.com/ulrikstrid/ocaml-oidc/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.8" & >= "2.5"}
```

Comment:
AFAICT you are doubled up on this constraint because your `dune-project` file mentions
- `lang` version `2.8` (https://github.com/ulrikstrid/ocaml-oidc/blob/master/dune-project#L1)
- `dune` constraints `2.5` (https://github.com/ulrikstrid/ocaml-oidc/blob/master/dune-project#L28)

You might want to drop the constraint and let dune infer its own version constraint.

Reply:
Thanks!